# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/pdisk/hotfix-makefile.patch
# Copyright (C) 2004 - 2025 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- pdisk-pdisk-10/makefile.vanilla	2025-06-28 18:22:12.701040351 +0200
+++ pdisk-pdisk-10/makefile	2025-06-28 18:34:09.561572523 +0200
@@ -11,7 +11,6 @@
 DOCS= \
 	HISTORY \
 	README \
-	$(MAN_PAGE) \
 	$(MAC_DOC)
 
 SERVER_README = \
@@ -91,7 +91,8 @@
 	io.o \
 	media.o \
 	pathname.o \
-	util.o
+	util.o \
+	cmdline.o
 
 UNIX_OBJECTS = \
 	pdisk.o \
@@ -101,6 +102,7 @@
 
 CVT_OBJECTS = \
 	cvt_pt.o \
+	dump.o \
 	$(COMMON_OBJECTS)
 
 
@@ -141,6 +140,17 @@
 cvt_pt: $(CVT_OBJECTS)
 	cc -o cvt_pt $(CVT_OBJECTS)
 
+doc_prepare:
+	mkdir doc
+	cp $(DOCS) doc
+
+install: all doc_prepare
+	install -d -m 755 /sbin
+	install -d -m 755 /usr/man/man8
+	install -s pdisk /sbin
+	install -s cvt_pt /sbin
+	install -m 444 pdisk.8 /usr/man/man8
+
 tags:	$(MAC_SOURCE) $(UNIX_SOURCE)
 	ctags $(MAC_SOURCE) $(UNIX_SOURCE)
 
@@ -212,6 +214,7 @@
 pdisk.o:	pdisk.c pdisk.h io.h partition_map.h pathname.h errors.h dump.h validate.h version.h util.h
 util.o:		util.c version.h util.h
 validate.o:	validate.c validate.h deblock_media.h pathname.h convert.h io.h errors.h
+cmdline.o:	cmdline.c
 
 
 #

